src/globalCallables.ts

```diff
@@ -250,29 +250,15 @@ let runtimeFunctions: GlobalCallable[] = [{
         type: DynamicType.instance,
         isOptional: false
     }]
-}, {
-    name: 'Run',
```

Comment:
This was duplicated -- I changed the function below to have a union type for teh param